src/api/gamestate.cc

```diff
@@ -53,7 +53,7 @@ CStateFunctionType ConvertStateFunctionType(mahjong::StateFunctionType func) {
     case mahjong::StateFunctionType::kGameEnd:
       return kGameEnd;
   }
-  
+
   // This should never be reached (unless we forget to add a new state function)
   return kError;
```

Comment:
The compiler should complain if we add a new state type and so we shouldn't need this additional code. The compiler might even be optimizing it out since there are no paths that hit it.

Reply:
@aliceofhartley ah! should i just remove this then?